### PR TITLE
Feature/rlp 1089/format fix

### PR DESCRIPTION
**Pull request by JONAF2103**
This PR aims to fix format issues, basically we apply prettier plugin with rules and also we added the husky lib to run prettier on pre-commit hook.

### CODE REVIEW

File: .huskyrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "hooks": {
+    "pre-commit": "npm run prettier"
+  }
+}
```
Comment (mauricioirace) [inline .huskyrc]: Missing EOL

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+.prettierignore
+generateImages
+package.json
+package-lock.json
+prettierrc.json5
+README.md
+tsconfig.json
+tslint.json
+node_modules
+coverage
```
Comment (mauricioirace) [inline .prettierignore]: Missing EOL

**Review by mauricioirace — APPROVED** [review]
LGTM! Just add the EOLs at the end of those files

File: .husky/pre-commit [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
```
Comment (mauricioirace) [inline .husky/pre-commit]: What would happen if I'm not using nvm?

Comment (JONAF2103) [reply .husky/pre-commit]: Good question, i will add a verification first

**Review by mortelli — APPROVED** [review]
superb, thanks for this. 

tested the hook, seems to work OK

**Pull request merged**